src/Console/Application.php

```diff
@@ -76,9 +76,7 @@ private function loadUserConfiguration(ContainerBuilder $container): void
         $configFile = $homeDirectory . '/.release-tool/auth.yml';
 
         if (!file_exists($configFile)) {
-            throw new RuntimeException(
-                sprintf('The file %s needs to exist and contain a GitHub access token', $configFile)
-            );
+            exit(sprintf('The file %s needs to exist and contain a GitHub access token.' . PHP_EOL, $configFile));
```

Comment:
I feel we should allow the file to be created when it does not exist.

Reply:
I'm afraid that just creating the file doesn't help that much, because a couple lines down the config file is being processed and afterwards the code is expecting that `$processedConfiguration['github']['token']` is available.

Maybe adding a `release-tool config credentials.github XXXXXXXXXXXXXXXXXXXXXXX` command, just like composer has, will offer a nice developer experience. In that case you can keep the error message with the exit and just refer to the `config` command to properly configure the credentials.

Reply:
I agree. I meant to interactively generate the file and ask for the token.
But maybe directing the user to another command is cleaner.